### DISCUSSION TITLE
Reset Auswahl von menuconfig?

### DISCUSSION THREAD

**Discussion by cbs38139** [Freetz-NG/freetz-ng#918] [Allgemein] [answered]
Ich habe am Wochenende erfolgreich ein image erstellt und auf meine Fritz!Box aufgespielt (echt cool!).

In meiner Begeisterung habe ich mich gleich an die Konfiguration eines zweiten image gemacht und wild Pakete dazugefügt und rausgenommen, erstellt und versucht aufzuspielen. Dieses zweite image konnte ich zwar ohne Probleme aufspielen, aber die Fritz!Box hing danach in einer blinkenden Endlos-Schleife und ging nicht mehr ins Netz.

Kein Problem, ich hatte noch das erste image und hab dieses wieder aufgespielt und Fritz!Box lief wieder.

Mein Problem: ich habe die Konfiguration des ersten image nicht gespeichert. Es war aber aber einfach die Standard-Konfiguration. Kann ich menuconfig wieder auf Anfang zurückstellen (und wie)?

Ich möchte dann nur ein einzelnes Paket hinzufügen und das image ausprobieren.

Danke!

**Comment by fda77** [comment, marked as answer]
In der Date `.config` werden alle Einstellungen gespeichert, man kann sie einfach löschen. Beim Erstellen eines Images wir sie dort auch hineingepackt und kann mit "tar" entpackt werden

**Reply by cbs38139** [reply]
Vielen Dank! Noch eine naive Frage: in welchem Ordner liegt die .config?

**Reply by fda77** [reply]
Im Hauptverzeichnis des Checkouts, zb `~/freetz-ng/.config`

**Reply by cbs38139** [reply]
Gefunden. Neues image generiert und läuft :-) Vielen Dank!
